docs/gallery/static/binaural_synthesis.ipynb

```diff
@@ -0,0 +1,469 @@
qV3hspQ8WRa+Q0cuRI9Ho9b7/9NitWrCAkJKRQcUiyaKOvvvoKgIEDB1psnzJlCl27dgVg9OjRqNVq3nnnHbKysoiIiLCYb0qj0bBw4UJmzZpFv379cHFxoUuXLrnqfJzZ8zV84okn+PDDD/noo49YtWoVlStXZsyYMTz77LPmY1577TUyMjKYOXMmycnJNGrUiMWLF1s8W/XBBx8wd+5cBg8ejEqlon379sU/R1YJ+euvvxg0aJD59YIFCwDo0qULU6dOtTi2UaNGLFy4kBEjRqBWq3n55ZdLM1S7VJj7SE4dOnRAURSmTJmCWq2mffv2pRLno8TermFh7iM5laffo4LuD4W5P+bUu3dvDAYDI0aMYPHixTRs2LBU3kdZKo5c40FjxozBYDAwaNAgVqxYQXBwcIFxqBRFUYrwPoQQQgghxCNMnlkUQgghhBD5kmRRCCGEEELkS5JFIYQQQgiRL0kWhRBCCCFEviRZFEIIIYQQ+ZJkUQghhBBC5EuSRSGEEEIIkS9JFoUQQgghRL4kWRRCiEfc1KlTGTNmTFmHIYQop2QFFyFEuTN16lS+++67XNu3bNlC1apVyyAi+5aSkoKiKHh4eNhcx4oVK9i3bx//+c9/LLbfuHGDbt26sXHjRsLCwooaqhDCDsna0EKIcqlly5ZMnjzZYpuPj0+u47Kzs3FwcCitsOySu7t7WYcghCjHJFkUQpRLDg4O+Pn55do+YMAAatasiVar5X//+x+1atVixYoVREVFsXjxYo4fP46LiwvNmzdnzJgxeHt7A5Cens6sWbPYs2cPrq6u9O3bl59++omwsDBzF254eDgffvghbdu2NZ+vbdu2jBkzhq5duwIQExPDwoULOXDgAGq1mkaNGjF27FgqV64MGFtFk5OTadSoEZ999hk6nY5OnToxZswYtFrjLTkrK4vly5ezY8cOEhISqFSpEq+//jrdu3enZ8+ePP/88/Tt29ccw9mzZ+nTp0++Laumc86fP998jUJDQ3F0dOSbb77BwcGBXr16MXDgwCJ/Lvm1+i5fvpzw8PAi1y+EKH3yzKIQ4pGzbds2tFotq1evZsKECSQnJ/P2228TFhbGp59+yuLFi0lISODdd981l1m0aBHHjh1j/vz5LF26lKNHj3L27FmrzqvT6Rg2bBiurq6sWrWK1atX4+rqyrBhw8jOzjYfd+TIEa5du8aKFSuYOnUqW7duZevWreb9U6ZMYceOHYwdO5Yvv/ySf/3rX7i6uqJSqejWrZvFsQBbt26lSZMmVnXBf/fdd7i4uLBu3TqGDRvGqlWrOHDggFXvNy9jx45l+/bt5p9//vOfVKhQgZCQkCLXLYQoG9KyKIQol3755Rdat25tft2yZUvmzJkDQNWqVRkxYoR536pVqwgLC2PIkCHmbZMnT+a5557j8uXL+Pv788033/DBBx/QrFkzwNhC1rlzZ6ti2rlzJwaDgUmTJqFSqQBj4te2bVuOHj1KixYtAPD09OSdd95Bo9EQEhLC3/72Nw4dOkTPnj25fPkyu3btYunSpTRv3hyAKlWqmM/RtWtXVqxYwcmTJ6lfvz46nY7t27czcuRIq2INDQ1lwIABAFSrVo1NmzZx+PBhc4x5iYqKsrjmAA8+9u7u7m7u9v7xxx/5+uuvWbp0aZ6twEKI8kGSRSFEudS0aVMmTJhgfu3i4mL+d926dS2OPX/+PEeOHMmV6ABcu3aNzMxMsrOzqV+/vnm7l5cXwcHBVsV0/vx5rl27xtNPP22xPSsri2vXrplf16hRA41GY37t5+dHVFQUAOfOnUOj0dC0adM8z+Hv70+rVq349ttvqV+/Pj/99BPZ2dl06NDBqlhDQ0MtXvv5+ZGQkPDQMsHBwfz73/+22BYbG5tn9/WZM2eYPHky48aNo1GjRlbFJoSwL5IsCiHKJRcXl3y7XXMmjmB8HrF169YMHz4817F+fn5cvXq1UOdUqVS5WtJ0Op3532lpadSpU4fp06fnKptz8I3p2cSc9RoMBgCcnJwKjKNHjx5MnjyZ0aNHs3XrVjp27Iizs3Oh3sPDYihocgwHB4dc1zxn0msSFxfHmDFj6NGjBz169LAqLiGE/ZFkUQjxyAsLC+PHH38kMDAwV5IExm5erVbLyZMnCQgIACApKYkrV65YtPD5+PgQFxdnfn3lyhUyMjLMr+vUqcOuXbvw8fGxeQRyrVq1MBgMHD161NwN/aBWrVrh4uLCV199xa+//sonn3xi07lKQmZmJmPGjCE4OJhRo0aVdThCiGIgA1yEEI+8l156iaSkJCZOnMipU6e4du0av/32G9OmTUOv1+Pq6kr37t1ZtGgRhw8fJioqiqlTp6JWW94iw8PD2bRpE2fOnOGvv/5i1qxZFsnns88+i7e3N2PGjOH48eNcv36dI0eOMG/ePG7dulWoWCtXrkyXLl14//332bt3r7mOXbt2mY/RaDR06dKFpUuXUq1aNRo0aFA8F6oYzJw5k1u3bjFu3Dju3LlDXFwccXFxFgN8hBDli7QsCiEeef7+/qxevZolS5YwdOhQsrKyCAwMJCIiwpwQjhgxgrS0NEaNGoWbmxt9+vQhJSXFop5Ro0Yxbdo03nrrLfz9/RkzZgynT58273d2dmblypUsWbKEcePGkZaWhr+/P82aNcPNza3Q8b777rssXbqU2bNnk5iYSEBAAP369bM4pnv37qxdu9Y8ZY+9OHbsGHFxcbz44osW22XqHCHKL1nBRQgh8jFgwACLeRbtyfHjx3n77bfZtm0bvr6+ZR2OEOIRJi2LQghRjmRlZXHnzh1WrlxJhw4dJFEUQpQ4eWZRCCHKkR07dtC1a1eSk5PzHN0thBDFTbqhhRBCCCFEvqRlUQghhBBC5EuSRSGEEEIIkS9JFoUQQgghRL4kWRRCCCGEEPmSZFEIIYQQQuRLkkUhhBBCCJEvSRaFEEIIIUS+JFkUQgghhBD5+n80sDT0iFmdXAAAAABJRU5ErkJggg==",
```

Comment:
sikersten and I came to the conclusion that we want the code to be as simple as possible. I feel like adding an indication for that would go against this. But it might also help with the context.